src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -109,6 +138,15 @@ def get_tags(
 ) -> Sequence[Mapping[str, str | bool]]:
     """Get tag keys and values for block kit"""
     fields = []
+    if not tags:
+        tags = set()
+
+    # XXX(CEO): context is passing tags as a list of tuples from self.event.tags
```

Comment:
This is definitely a bit of a hack but it seemed like too deep a rabbithole to go into in this PR - [here](https://github.com/getsentry/sentry/blob/master/src/sentry/notifications/notifications/rules.py#L188) is where it's being set in some cases.